doc/index.md

```diff
@@ -104,6 +104,9 @@ interface ISdkConfig {
         };
         videoDeviceId?: string | null;
         audioDeviceId?: string | null;
+        micAutoGainControl: ConstrainBoolean;
+        micEchoCancellation: ConstrainBoolean;
+        micNoiseSuppression: ConstrainBoolean;
```

Comment:
NIT: `micNoiseSuppression?: ConstrainBoolean | boolean;` for all three (missing `?` and `| boolean`).

Reply:
These also need to be added under `index.md#defaults` as `index.md#defaults.micAutoGainControl`, `index.md#defaults.micEchoCancellation`, and `index.md#defaults.micNoiseSuppression`.

Reply:
Good catch. I'm losing my mind apparently. So many places to doc.